src/controllers/user.js

```diff
@@ -59,11 +59,18 @@ export const getAll = async (req, res) => {
 }
 
 export const updateById = async (req, res) => {
-  const { cohort_id: cohortId } = req.body
-
-  if (!cohortId) {
-    return sendDataResponse(res, 400, { cohort_id: 'Cohort ID is required' })
+  if (
+    req.body.firstName &&
```

Comment:
Minor but I'd suggest reversing the logic here - 

``
if requst is not valid
  return error

//do stuff when request is valid
```

That formatting would be more conventional and consisten with other validation checks in the controller.

Reply:
done